test/batchReceiver.spec.ts

```diff
@@ -213,4 +213,157 @@ describe("ReceiveBatch from Queue/Subscription", function(): void {
 
     await receivedMsgs[0].complete();
   });
+
+  it("Abandoning a message 10 times, you get it no more. You can then only get it from the dead letter queue", async function(): Promise<
```

Comment:
I wrote the above line in the github issue so that the text was self explainatory to whoever was writing the tests. We can improve it here. How about this:

> Message abandoned more than maxDeliveryCount goes to dead letter queue